src/main/java/com/microsoft/aad/msal4j/ServerSideTelemetry.java

```diff
@@ -17,7 +16,7 @@
 
     private final static Logger log = LoggerFactory.getLogger(ServerSideTelemetry.class);
 
-    private final static String SCHEMA_VERSION = "2";
+    private final static String SCHEMA_VERSION = "5";
```

Comment:
Should LastRequest also be bumped up to version 5? I think keeping current request and last request in sync could be important from a kusto query perspective.

Reply:
It turns out that LastRequest hasn't changed between schema versions 2 and 5 so no worries there, and in the latest commit I made all of the non-regional telemetry changes to more thoroughly follow schema version 5.